src/@components/JoinPage/UserProfilePage/ProfileJender/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { St } from "./style";
+
+interface jenderTypes {
+  isSelected: string;
+  setIsSelected: (jender: string) => void;
+}
+
+export default function ProfileJender(props: jenderTypes) {

```

Comment:
⚡반영 필요⚡--> `commit suggestion`

```suggestion
interface ProfileJenderProps {
  isSelected: string;
  setIsSelected: (jender: string) => void;
}

export default function ProfileJender(props: ProfileJenderProps) {
```